Assets/Scripts/Hazard.cs

```diff
@@ -1,21 +1,29 @@
-﻿using System.Collections;
+﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.SceneManagement;
 
 public class Hazard : MonoBehaviour
 {
+    [SerializeField]
+    private float audioDelay = 0.5f;
     private AudioSource audioSource;
     private void Start()
     {
         audioSource = GetComponent<AudioSource>();
     }
 
     private void OnTriggerEnter2D(Collider2D collision)
+    {
+        StartDeath(collision);
```

Comment:
I know "collision" is the template parameter name, but do you think it reads more clearly if it were named something like `objectCollidedWith`?

Reply:
I guess it's a `Collider`, so maybe it should be `otherCollider`. Just thoughts.